libtenzir/builtins/formats/syslog.cpp

```diff
@@ -149,8 +221,10 @@ auto impl(generator<std::optional<std::string_view>> lines,
     if (rows() >= defaults::import::table_slice_size
         or last_finish + defaults::import::batch_timeout < now) {
       last_finish = now;
-      for (auto&& slice : finish()) {
-        co_yield std::move(slice);
+      if (auto slice = finish(); not slice) {
+        co_return;
+      } else if (slice->rows() > 0) {
+        co_yield std::move(*slice);
```

Comment:
There is a small bug hidden here: If the last message was a multi-line one and we either reach the timeout or have collected `defaults::import::table_slice_size` messages, then we will emit it without its continuation, and will produce an extra `syslog.unknown` message instead.

Reply:
Unresolving, as it seems that this is not fixed yet? Perhaps I'm mistaken, please let me know if I'm missing something.

Reply:
Should really be fixed now.